App.tsx

```diff
@@ -1,20 +1,48 @@
-import { StatusBar } from 'expo-status-bar';
-import { StyleSheet, Text, View } from 'react-native';
+import {NavigationContainer} from '@react-navigation/native';
+import {BottomTabNavigationOptions, createBottomTabNavigator} from '@react-navigation/bottom-tabs';
+import AllMuseumsScreen from "./screens/AllMuseumsScreen";
+import SearchMuseumScreen from "./screens/SearchMuseumScreen";
+import FavouriteMuseumsScreen from "./screens/FavouriteMuseumsScreen";
+import React from "react";
+import Ionicons from '@expo/vector-icons/Ionicons';
+import { Fontisto } from '@expo/vector-icons';
 
+const Tab = createBottomTabNavigator();
 export default function App() {
-  return (
-    <View style={styles.container}>
-      <Text>Open up App.tsx to start working on your app!</Text>
-      <StatusBar style="auto" />
-    </View>
-  );
+    return (
+        <NavigationContainer>
+            <MyTabs/>
+        </NavigationContainer>
+    );
 }
 
-const styles = StyleSheet.create({
-  container: {
-    flex: 1,
-    backgroundColor: '#fff',
-    alignItems: 'center',
-    justifyContent: 'center',
-  },
-});
+function MyTabs() {
+    return (
+
+        <Tab.Navigator>
+            <Tab.Screen name="All" component={AllMuseumsScreen} options={homeIcon()}/>
```

Comment:
The `homeIcon()` is not very clear in my opinion, because in reality we should pass an options object. The same applies to the next two tabs. 

Additionally, there is nothing bad about using the `props` object, but in general I think that destructuring is slightly better, because it allows us to get only the props that we need, and simplifies the code when we work on bigger components

```suggestion
            <Tab.Screen name="All" component={AllMuseumsScreen} options={{tabBarIcon: ({color}) => <Ionicons name="home" size={20} color={color} />}}/>
```